### ISSUE TITLE
ID

### ISSUE THREAD

**Issue by yezz123** [twintproject/twint#959]
How can I get the ID's of the followers instead of the username?

**Comment by himanshudabas** [comment]
It hasn't been implemented in the library yet.
To be honest I haven't gotten to that part yet. So, I am not yet sure if it's even possible to get the **User IDs** from twitter as of now. If I am not wrong, the current implementation of getting the lost of followers uses the _mobile version_ to fetch the **followers list**, which might not have the User IDs of the respective followers in the HTML source.

**In short, Currently It's not possible.**

Will update on this thread _if_ and _when_ I find a way to implement this.
